README.md

```diff
@@ -5,13 +5,6 @@ SFT - Selective Forwarding TURN
 
 * Ubuntu 18.04
 
-### Terminology
-
-__Server:__ an instance hosting the SFT service as well as a local proxy in front of it.
-
-__Load balancer:__ a different instance from *servers*, that distributes new incoming conference calls across
-*servers* during the *signaling* procedure.
-
```

Comment:
Why remove this?

Reply:
We no longer have load balancers in our design, this is outdated information.